### DISCUSSION TITLE
Ability to use operators on aggregated fields

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5720] [Feature Requests]
> It would be very useful to have the ability to utilise some of the standard operators in a query (eg, (does not) contains, (not) empty) on an aggregated field.
> 
> This is the current scenario we are trying to resolve. Our Trait data is sitting in the collectionobjectproperty table. As such, data gets only entered when values are present on data entry. For our egg collection, records mostly have the traits ‘Clutch size’ and ‘Nest kept y/n’ (see screenshot).
> 
> The user would like to identify the collection objects where the trait ‘Clutch size’ does not exist. We have not been able to create a query that does this for us. The closest we have come is to utilise the Trait (aggregated) field and then do a check on the string in Excel. It would be beneficial if an operator could be placed on the aggregated field to indicate a return of all records where ‘Clutch Size’ isn’t contained within the string.
> 
> In other instances, I have wanted to only return records where the aggregation is not null (eg, only give me those events where I have permits), however, I had to pull in the permit field separately and put the condition on that field.
>
> ![image](https://user-images.githubusercontent.com/37256050/214189623-64ed5891-3b93-4e26-a3b7-1124c6ae1931.png)

**Requested By:** Corinna P at CSIRO ([Asana Link](https://app.asana.com/0/1201972579963460/1203764884727925/f))

**Comment by maxpatiiuk** [comment]
See also #318
